RickLeinecker/Fortuna
# Initial test setup and initial test for user model

**Description**
* Did initial changes to files to support testing using Jest. 
  * Mainly just splitting the functionality of the Express server application from the Server Listener and the connection to the DB.
  * Edited the Babel settings to ignore the test files.
* Added an initial test for the user model to ensure `npm test` works.
* Did updates to the PR template to simplify things
* Updated the `README.md` to update some npm commands

Resolves #129 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Checkout branch.
2. Delete `lib`, `node_modules` directories and `package-lock.json`
3. `npm i`
4. `npm run build && npm run devserver` to check if the server functionality is still good.
5. Close `devserver`
6. `npm test` and see if the initial tests pass.

**Final Checklist:**
Go down the list and check them off.
